server/query.js

```diff
@@ -126,11 +126,11 @@ const Query = {
     async getQuestions(req, res, aid) {
         await query(req, res, 'SELECT * FROM question WHERE assignment_id=$1', [aid])
     },
-    async createQuestionSubmission(req, res, uid, aid, num) {
-        await query(req, res, 'INSERT into questionsubmission (student_id, assignment_id, question_number) values ($1, $2, $3)', [uid, aid, num])
+    async getAssignmentSubmission(req, res, id, aid) {
+        await query(req, res, "SELECT * FROM assignmentsubmission WHERE student_id=$1 AND assignment_id=$2", [id, aid])
     },
-    async getAssignmentSubmissions(req, res, id) {
-        await query(req, res, 'SELECT * FROM assignmentsubmission WHERE student_id=$1', [id])
+    async getAssignmentSubmissions(req, res, id, aid=null) {
```

Comment:
Do we intend on using the aid parameter here?

Reply:
no, relic from an old thing